eppoclient/client.go

```diff
@@ -82,11 +83,11 @@ func (ec *EppoClient) GetJSONAssignment(flagKey string, subjectKey string, subje
 
 func (ec *EppoClient) getAssignment(flagKey string, subjectKey string, subjectAttributes Attributes, variationType variationType) (interface{}, error) {
 	if subjectKey == "" {
-		panic("no subject key provided")
+		return nil, fmt.Errorf("no subject key provided")
 	}
 
 	if flagKey == "" {
-		panic("no flag key provided")
+		return nil, fmt.Errorf("no flag key provided")
 	}
```

Comment:
while this is extremely undesirable inputs; they will return the default value and the user can handle the error safely. the panic made sense during previous implementations of the SDK before we added default values.